simulator/cluster_compute_resource_test.go

```diff
@@ -83,6 +87,35 @@ func TestClusterVC(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// Enable DRS and HA for the cluster
+	clusterSpec := types.ClusterConfigSpecEx{
+		DrsConfig: &types.ClusterDrsConfigInfo{
+			Enabled:           types.NewBool(true),
+			DefaultVmBehavior: types.DrsBehaviorFullyAutomated, // Set DRS to fully automated
+		},
+		DasConfig: &types.ClusterDasConfigInfo{
+			Enabled: types.NewBool(true),
+		},
+	}
+
+	task, err := cluster.Reconfigure(ctx, &clusterSpec, true)
+	require.NoError(t, err)
```

Comment:
I think it would be good to avoid using require here as well, I am not sure testify is widely used on the project